integration-tests/kyuubi-kubernetes-it/pom.xml

```diff
@@ -68,6 +68,12 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>io.fabric8</groupId>
+            <artifactId>kubernetes-httpclient-okhttp</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
why should we need it?

Reply:
Fixed `Minikube` use okhttp-client.

Reply:
I don't get your point, it should work w/o introduce extra deps.

Reply:
Add comment in parent `pom.xml`,       
> according to kubernetes-client MIGRATION-v6.md, we should include this dependency
   for doing any customization to OkHttp clients.
   https://github.com/fabric8io/kubernetes-client/blob/master/doc/MIGRATION-v6.md#okhttp-httpclient

Reply:
You already defined it in the kyuubi-server module as compile scope, why should define it again?

Reply:
Right, i misunderstand XD, removed.